xcell/mappers/utils.py

```diff
@@ -77,3 +79,49 @@ def get_beam(nside, beam_info):
     else:
         raise NotImplementedError("Unknown beam type.")
     return beam
+
+
+def _get_custom_wf(wf_info):
+    field = wf_info['field']
+    file = wf_info['file']
+    windowfuncs = pd.read_csv(file, comment='#')
+    wf = interp1d(np.array(windowfuncs['ell']),
+                  np.array(windowfuncs[field]),
+                  fill_value='extrapolate')
+    return wf
```

Comment:
This one seems very specific. I think this should probably live in the very specific mapper that needs it, while the more general one can live in the base mapper

Reply:
https://github.com/xC-ell/xCell/blob/830b3f246ce96fb57de03f4b5ec9c6831da52cd9/xcell/mappers/utils.py#L84-L85

https://github.com/xC-ell/xCell/blob/830b3f246ce96fb57de03f4b5ec9c6831da52cd9/xcell/mappers/mapper_LenzCIB.py#L18-L25